tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -221,7 +221,6 @@ pub mod pallet {
 			})
 		}
 
-		#[cfg(feature = "development")]
```

Comment:
I assume this removal is quiet safe - runtime is reachable only through trusted calls.